pkg/cmd/exec/exec.go

```diff
@@ -8,44 +8,42 @@ import (
 	"strings"
 	"sync"
 
+	"github.com/reposaur/reposaur/pkg/cmdutil"
 	"github.com/reposaur/reposaur/pkg/detector"
 	"github.com/reposaur/reposaur/pkg/output"
 	"github.com/reposaur/reposaur/pkg/sdk"
-	"github.com/rs/zerolog"
 	"github.com/spf13/cobra"
 )
 
 type Params struct {
 	namespace    string
 	outputFormat string
-	loggerLevel  string
-	loggerFormat string
 	policyPaths  []string
 }
 
 var cmd = &cobra.Command{
-	Use:   "reposaur",
+	Use:   "exec",
 	Short: "Executes a set of Rego policies against the data provided",
 	Long:  "Executes a set of Rego policies against the data provided",
 }
 
-func NewCommand() *cobra.Command {
+func NewCommand(f *cmdutil.Factory) *cobra.Command {
```

Comment:
is this factory being used? cannot find where..

Reply:
Yes, here: https://github.com/reposaur/reposaur/pull/13/files/7c7bdb74f820afdeb70c03e2645c78750fb52d7c#diff-0b23fbb550d65237bf314a33bea972602305a7bd7572b94a191fa732cf8ae5b7R44

Reply:
L44 on the new changes

Reply:
sry was #blind